poanetwork/token-wizard
# (Bug) Keep data between the steps (Step 3)

This is an integration PR for the step 3.

How to test it and what to test:
- Go to the home
- Create a crowdsale
- In the first step, press button "New crowdsale"
- Select the type of strategy
- Go to the second step
- Go back to step 1
- Should keep the selected strategy
- Refresh the step 1 page
- Should keep the selected strategy
- Got to the second step
- Complete the form
- Refresh the page, should keep the data
- Add reserved tokens
- Refresh the page, should keep the data
- Got to the third step
- Complete the form
- Refresh the page, should keep the data
- Go to the home, and press button "New crowdsale"
- The selected strategy must be the default, "WHITELIST WITH CAP"
- Go to the second step
- The form must be empty
